### PR TITLE
Anthony_React_Testing_Vitest_simple&snapshot

### PR DESCRIPTION
**Pull request by O-Reed**
The description for a test should:

Be clear and concise.
Describe the behavior being tested.
Follow a pattern, such as should, when, or expect, to explain the expected outcome.

### CODE REVIEW

File: src/__test__/UsernameDisplay.test.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { render, screen } from "@testing-library/react";
+import UsernameDisplay from "../components/UsernameDisplay";
+
+describe("UsernameDisplay", () => {
+  it("should render username", () => {
+    const result = render(<UsernameDisplay username="anthony" />);
+    // expect(result.container).toMatchSnapshot();
+    expect(screen.getByText(/anth/)).toBeInTheDocument();
+  });
+});
```
Comment (O-Reed) [inline src/__test__/UsernameDisplay.test.jsx:11]: Great job testing the basic use case. Can you add a test for when the username prop is null or empty? This will help cover edge cases.

Comment (O-Reed) [reply src/__test__/UsernameDisplay.test.jsx:11]: Thanks for the feedback! I'll add a test case for null and empty usernames. Also, I’ll update the code to use getByRole for better accessibility.

**Review by O-Reed — COMMENTED** [review]
Added tests for the UsernameDisplay component. The test now covers the basic rendering of the username prop.

**Comment by O-Reed** [comment]
Explain why something is done, not just what is done.
Avoid obvious comments that don't add value.

**Comment by O-Reed** [comment]
Explain why something is done, not just what is done.
Avoid obvious comments that don't add value.
